cypress/test/layout/GenericWideLayout.cy.js

```diff
@@ -0,0 +1,32 @@
+import { mount } from 'cypress/react';
```

Comment:
Delete this component test file as it is no more in use inside codebase.

Reply:
done